### DISCUSSION TITLE
Spring Data Mongodb ignored by agent

### DISCUSSION THREAD

**Discussion by jeusdi** [open-telemetry/opentelemetry-java-instrumentation#11187] [Q&A]
I've created my lambdas attaching ADOT(Aws distribution for opentelemetry, based on java agent) layer.

When my function is reached it's generating two spans: one span for lambda invocation and another one for function code.

Inside my function code, I'm reaching a mongodb using Spring Data Mongodb.

According to documentation, mongodb instrumentation is automatically added by agent. Nevertheless, no mongodb span is attached on trace.

Any ideas about how could I attach mongodb spans?

**Comment by laurit** [comment]
Could you provide a minimal sample application that does not use lambda and reproduces this issue.
